src/components/previews/index.tsx

```diff
@@ -18,4 +19,5 @@ export const Previews = {
   [PREVIEW.SETTINGS]: () => <SettingsPreview />,
   [PREVIEW.NOW_PLAYING]: () => <NowPlayingPreview />,
   [PREVIEW.SERVICE]: () => <ServicePreview />,
+  [PREVIEW.DEVICE]: () => <div>SILVER OR BLACK</div>,
```

Comment:
Let me get you svgs for these. I have a few iPod svgs I created that we can use to show the current colors

Reply:
Updated, it's not an SVG because I don't want to be Designer Mayank right now and create it from scratch. Feel free to update the png to an svg with your figma file!